README.md

```diff
@@ -1,35 +1,148 @@
 GodTools
-=======
-
-Version 5.7.9
+========
 
 - [Requirements](#requirements)
-- [Architectural Pattern](#architectural-pattern)
+- [Programming Guide](#programming-guide)
 
 ### Requirements
 
-- Xcode 13
-- iOS 11+
+- Xcode 13.4.1
 - Bundler
 - Cocoapods
 - Fastlane
 
-### Architectural Pattern
+### Programming Guide
+
+- [Conventions](#conventions)
+- [Architecture](#architecture)
+
+#### Conventions
+
+- [Classes](#classes)
+
+
+#### Classes
+
+- Class / Struct attributes should always be declared with the type.
+- Class level attributes should also be grouped by constant vs instance as well as by access level and mutability. Where constants are at the top and grouped by private first, then internal, then public.  Instance variables follow constants and are grouped by private first, then internal, then public.   
+
+#### Architecture
+
+![alt text](ReadMeAssets/clean_architecture.png)
+
+- [Summary](#summary)
+- [Presentation Layer](#presentation-layer)
+- [Domain Layer](#domain-layer)
+- [Data Layer](#data-layer)
+- [Coordinator](#coordinator)
+
+
+#### Summary
+The architectural structure for the GodTools app can be summarized as following a Clean Architecture Pattern (Presentation Layer, Domain Layer, an Data Layer), along with a Coordinator Pattern (Navigation decisions and logic), and SOLID principles(https://www.geeksforgeeks.org/solid-principle-in-programming-understand-with-real-life-examples/) .
+
+The purpose of this pattern is to create a clear separation of concerns and responsibilities.
+
+#### Presentation Layer
+
+The presentation layer makes up the Views and ViewModels.  In the GodTools app Views and ViewModels are organized by Feature in the Features folder.  Below will explain how presentation files are named and organized and conventions for Views and ViewModels.
+
+- [File Naming and Organization](#file-naming-and-organization)
+- [Views](#views)
+- [ViewModels](#viewmodels)
+
+##### File Naming and Organization
```

Comment:
This section is a little hard to follow.  It might be easier to follow bullet points that mimic the structure with indentions?  Or a screenshot of some of the directories in Xcode for some of the examples you mention?  (Or both?).  Also would be good to mention if you want things sorted alphabetically (that's not something I've seen on other projects!).

📁**Features**
  - 📁 Lessons
   --   📁 LessonList
    -----  📄 LessonList.swift
    -----  📄 LessonListViewModel.swift
    -----  📁 Subviews
     --------   📄 LessonListTitleView.swift

^ not formatted well but just an idea to help cut down on some of the text here.

Reply:
I think a visual would be helpful here too.  I will make some updates here.